### DISCUSSION TITLE
Number of requests per second for one hour

### DISCUSSION THREAD

**Discussion by MHR00** [datalust/seq-tickets#2017] [Help] [answered]
![image](https://github.com/datalust/seq-tickets/assets/47424123/d89eded3-bb30-4561-afe5-174e3e84b97b)
Hi Dear , 

How to modify this query to check all request for one hour and show Most count of request for each service pre second . 

i want the result not include each second , just check one hour for each but response should only service and count .

BR.

**Comment by KodrAus** [comment]
Hi @MHR00 :wave:

If I understand correctly you'd like to show the max of the count of events by second, grouped by hour? I think you'd need to use a subquery to do this, which Seq doesn't currently support.

Depending on how you're consuming that data you may be able to use Seq's HTTP API to perform your query by second and then aggregate into hours on the client.

**Comment by MHR00** [comment]
Yes, that's exactly what we want to do.

Is it possible to explain the solution you mentioned more?

**Reply by liammclennan** [reply, marked as answer]
Hi @MHR00 ,

@KodrAus is on leave so I will do my best to help here. 

I believe he was suggesting that you query Seq, via [the API](https://github.com/datalust/seq-api), with a `group by time(1s)`. Then, within your client application, group those results by hour.
